solutions/devsprint-michel-ferreira-2/src/main/resources/db/migrations/V8__alter_table_parking_spot_event_add_vehicle_id.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE parking_spot_event ADD vehicle_id varchar(36) NOT NULL after event;
```

Comment:
Dica super importante:
Como estamos trabalhando com migration e queremos adicionar uma nova coluna Not Null, há algumas regras que precisamos seguir:
1 - Criamos a coluna como Nullable.
2 - Atualizamos os valores já existentes no BD com algum valor default ( Se isso não for possível, não será possível proseguir e será preciso pensar em outra estratégia)
3 - Atualizar a constraint de Not Null na nova coluna criada.